cli/src/cli_util.rs

```diff
@@ -1434,6 +1437,19 @@ to the current parents may contain changes from multiple commits.
         Ok(git_ignores)
     }
 
+    #[cfg(not(feature = "git"))]
+    pub fn base_attributes(&self) -> Result<Arc<GitAttributesFile>, ConfigGetError> {
+        Ok(Arc::new(GitAttributesFile::default()))
+    }
+
+    #[cfg(feature = "git")]
+    #[instrument(skip_all)]
+    pub fn base_attributes(&self) -> Result<Arc<GitAttributesFile>, ConfigGetError> {
+        Ok(Arc::new(GitAttributesFile::new(
+            &self.settings().git_settings()?.ignore_filters,
+        )))
+    }
+
```

Comment:
nit: use `cfg!` instead:
```rust
    #[instrument(skip_all)]
    pub fn base_attributes(&self) -> Result<Arc<GitAttributesFile>, ConfigGetError> {
        if cfg!(feature = "git") {
            Ok(Arc::new(GitAttributesFile::new(
                &self.settings().git_settings()?.ignore_filters,
            )))
        } else {
            Ok(Arc::new(GitAttributesFile::default()))
        }
    }
```

That way we always compile-test both branches and IDEs can better understand both branches too.